File: app/src/main/java/com/asuka/androidopensourcelibrarystudydemo/utils/Logger.java
package com.asuka.androidopensourcelibrarystudydemo.utils;

import android.os.Build;

import com.asuka.androidopensourcelibrarystudydemo.BuildConfig;

import timber.log.Timber;

public class Logger {
    public static void init(){
        if (BuildConfig.DEBUG){
            Timber.plant(new Timber.DebugTree());
        }
    }

    public static void d(String s){
        Timber.d(s);
    }

}

File: app/src/main/java/com/asuka/androidopensourcelibrarystudydemo/utils/Logger.kt
package com.asuka.androidopensourcelibrarystudydemo.utils

import com.asuka.androidopensourcelibrarystudydemo.BuildConfig
import timber.log.Timber
import timber.log.Timber.DebugTree

object Logger {
    fun init() {
        if (BuildConfig.DEBUG) {
            Timber.plant(DebugTree())
        }
    }

    fun d(s: String?) {
        Timber.d(s)
    }
}